src/SFA.DAS.Forecasting.Triggers/Startup.cs

```diff
@@ -101,7 +102,7 @@ public IServiceProvider Build()
             services.AddSingleton<ILevyForecastService, LevyForecastService>();
             services.AddSingleton<IPaymentForecastService, PaymentForecastService>();
             services.AddSingleton(typeof(IHttpFunctionClient<>), typeof(HttpFunctionClient<>));
-            //services.AddApplicationInsightsTelemetry(Configuration["APPINSIGHTS_INSTRUMENTATIONKEY"]);
+            // services.AddApplicationInsightsTelemetry(Configuration["APPINSIGHTS_INSTRUMENTATIONKEY"]);
```

Comment:
Should this be uncommented?

Reply:
no can probably be deleted as per [Microsoft](https://docs.microsoft.com/en-us/azure/azure-functions/functions-dotnet-dependency-injection) it should not be added